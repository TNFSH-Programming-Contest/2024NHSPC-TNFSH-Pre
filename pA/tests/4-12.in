318971667914752861 245452930814268136 999073076958842300
318971667914752861 245452930814268136 999073076958842300
