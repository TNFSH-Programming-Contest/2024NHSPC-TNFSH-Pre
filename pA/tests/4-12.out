736358792442804408
